samples/deployment/vmware/README.md

```diff
@@ -0,0 +1,66 @@
+# Setup Guide
+
+1. Create virtual env
+    ```bash
+    virtualenv -p python3.7 ./venv
+    source venv/bin/activate
+    pip install ansible pyvmomi
+    ```
+
+1. Install vsphere automation python sdk 
```

Comment:
Please, pay attention to upper-case letters in names. Here, it should be `vSphere Automation Python SDK`.

Reply:
fixed